pallets/loans/src/ptoken.rs

```diff
@@ -206,12 +206,10 @@ where
         let market = Self::ensure_active_market(underlying_id)?;
         let collateral_value = Self::collateral_asset_value(who, underlying_id)?;
 
-        // liquidity of all assets
-        let (liquidity, _) = Self::get_account_liquidity(who)?;
-
-        if liquidity >= collateral_value {
+        if let Ok(()) = Self::ensure_liquidity(who, collateral_value) {
```

Comment:
There is no need to change it because liquidity is used down here

Reply:
Okay, I will change it back